esphome/components/mitsubishi_itp/mitsubishi_uart-packetprocessing.cpp

```diff
@@ -111,6 +118,18 @@ void MitsubishiUART::process_packet(const SettingsGetResponsePacket &packet) {
   target_temperature = packet.get_target_temp();
   publish_on_update_ |= (old_target_temperature != target_temperature);
 
+  switch (mode) {
+    case climate::CLIMATE_MODE_COOL:
+    case climate::CLIMATE_MODE_DRY:
+      this->last_cool_setpoint_ = target_temperature;
+      break;
+    case climate::CLIMATE_MODE_HEAT:
+      this->last_heat_setpoint_ = target_temperature;
+      break;
+    default:
+      break;
+  }
+
```

Comment:
Shouldn't we also handle HEAT_COOL here as well?

Reply:
Ah, yes. My system doesn't send this (see above MHK conflict), but it should be set. Same logic problem applies though...